### PR TITLE
IV iron location expansion data

### PR DESCRIPTION
**Pull request by alibow**
All of these data updates should be ready for artifact building now :) 

Please let me know if you're having trouble accessing anything that you expected to have. Thanks!

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
Looks like some updates to the SAM model got included in this PR unintentionally

**Comment by alibow** [comment]
Ok, only relevant info should be present now. Sorry about that!

File: docs/source/concept_models/vivarium_iv_iron/concept_model.rst [hunk 1]
```diff
@@ -208,6 +215,59 @@ For GBD outcomes that do not have estimates available for the locations of inter
...
+
+As new locations are added to this model, there are several data values that need to be updated in the artifact in addition to standard GBD data updates. These calculations are performed in the `vivarium_research_iv_iron repository <https://github.com/ihmeuw/vivarium_research_iv_iron>`_  and include:
+
+- Weights used in calculation of the below values
+  
+  - `Calculation performed in this notebook <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/parameter_aggregation/Generate%20weights.ipynb>`_
+
+- Hemoglobin validation targets
+ 
+  - `Done for prevalence here <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/hgb_validation_targets/Generate%20validation%20targets.ipynb>`_. Note that the anemia prevalence .csv file is too large to store in git, so it will have to be generated locally. Remeber to delete it before commiting changes in order to avoid annoying git issues.
```
Comment (plinds) [inline docs/source/concept_models/vivarium_iv_iron/concept_model.rst:229]: Typo: remember

**Review by plinds — APPROVED** [review]

File: docs/source/concept_models/vivarium_iv_iron/concept_model.rst (hunk 1 above)
Comment (zmbc) [inline docs/source/concept_models/vivarium_iv_iron/concept_model.rst:229]: You might want to add the filepath to the `.gitignore` file in the repo ([see documentation](https://git-scm.com/docs/gitignore)) to make sure it is never accidentally committed!

Comment (alibow) [reply docs/source/concept_models/vivarium_iv_iron/concept_model.rst:229]: Good thinking :) will do, thanks

**Pull request merged**